infra/gcp/clusters/kubernetes-public/aaa/10-cluster-configuration.tf

```diff
@@ -158,3 +158,18 @@ resource "google_container_cluster" "cluster" {
     enabled = true
   }
 }
+
+provider "kubernetes" {}
```

Comment:
GKE cluster `aaa` auth. creds need to be passed to the provider. A good example : 
https://www.terraform.io/docs/providers/kubernetes/index.html#statically-defined-credentials.

Reply:
Yes, I forgot to change the title to `[WIP]` cause I'm gonna start discussion about it tomorrow. We currently are running terraform locally. With kubernetes provider we would have to use google console only because ingress traffic to `aaa` is restricted to locale one only.